### PR TITLE
added code for getting and deleting relationships

### PR DESCRIPTION
**Pull request by SergeoLacruz**
Needed this for my import program.

### CODE REVIEW

File: inventree/base.py [hunk 1]
```diff
@@ -256,3 +256,9 @@ class ParameterTemplate(InventreeObject):
     """ class representing the Parameter Template database model"""
 
     URL = 'part/parameter/template'
+
+
+class Related(InventreeObject):
```
Comment (SchrodingersGat) [inline inventree/base.py:261]: Why is this class here? The "PartRelated" Model already exists in "part.py"

File: inventree/part.py [hunk 2]
```diff
@@ -83,6 +83,10 @@ def getParameters(self):
         """ Return parameters associated with this part """
         return inventree.base.Parameter.list(self._api, part=self.pk)
 
+    def getRelated(self):
+        """ Return parameters associated with this part """
```
Comment (SchrodingersGat) [inline inventree/part.py:87]: Comment string is incorrect

File: inventree/part.py [hunk 3]
```diff
@@ -256,3 +260,9 @@ def add_related(cls, api, part1, part2):
             logging.warning("Related failed")
             ret = False
         return ret
+
+    @classmethod
+    def delete_related(cls, api, part):
```
Comment (SchrodingersGat) [inline inventree/part.py:265]: Doesn't the existing "delete" function provide this functionality already? What exactly is the function here?

Comment (SergeoLacruz) [reply inventree/part.py]: Yes, I did not see this. Thanks  for the quality control. I am just starting. Sorry for the inconvenience.

**Comment by SchrodingersGat** [comment]
Thanks for the contribution!

**Pull request merged**